### DISCUSSION TITLE
Text Padding

### DISCUSSION THREAD

**Discussion by cool-dev-guy** [fonttools/fonttools#3102] [Q&A] [answered]
I am working on a project and i need to add text gap and text height.

I have a ttf_font: `myfont.ttf`
To add text height and gap,
It can be done by adding padding to left-right and top-buttom of each character of a ttf_font.
- I am having errors with `fonttools`
- Can you guys provide any example code.

**Comment by justvanrossum** [comment]
Your problem description sounds more like something to be done at the typesetting level, rather than by modifying a font.

**Reply by cool-dev-guy** [reply]
But i have a script with auto wrapping text box,the only way to apply padding is to add padding in font characters.

**Reply by justvanrossum** [reply, marked as answer]
Fixing typesetting limitations through font modification is still a bad idea if you ask me.

**Reply by cool-dev-guy** [reply]
Thanks for the info :)
